### PR TITLE
Add support for Jasmine 3.0.0

### PR DESCRIPTION
**Pull request by dfederm**
Closes: #31 

I basically just took the latest boot.js, jasmine-html.js, and jasmine.css from jasmine, which is essentially all this repo seems like it is, more or less.

### CODE REVIEW

File: src/lib/html.jasmine.reporter.js [hunk 1]
```diff
@@ -140,18 +140,21 @@ jasmineRequire.HtmlReporter = function(j$) {
...
                 })
             ));
             var checkbox = find("input");
 
-            checkbox.checked = !env.catchingExceptions();
-            checkbox.onclick = onRaiseExceptionsClick;
+            // Back-compat with jasmine < 3.0.0
+            var stoppingOnSpecFailure = env.stoppingOnSpecFailure || env.catchingExceptions;
+
+            checkbox.checked = stoppingOnSpecFailure();
```
Comment (dfederm) [inline src/lib/html.jasmine.reporter.js]: This checkbox seemed backwards to me, so I inverted it

**Comment by konpikwastaken** [comment]
Waiting on PR merge & npm update eagerly :)

**Comment by dfederm** [comment]
I've emailed @taras42 about approving this change and pushing the package. I'll push a temp package with the fix later this weekend if needed and eventually I'll request a takeover of the abandoned package from npm support.

**Comment by dfederm** [comment]
I've temporarily uploaded this npm package: https://www.npmjs.com/package/karma-jasmine3-html-reporter.

Also, I've started the process to request ownership of an abandoned package with the npm support team and according to the documentation, if @taras42 doesn't respond in like 4 weeks, I should be able to take ownership. If I'm mistaken about them being inactive (it's currently the weekend after all) and they do respond and agree to update the package, then even better :)

**Comment by andy-armstrong** [comment]
@dfederm Thank you so much for taking over this invaluable plugin! I have a quick question for you...

I've been running into problems with this plugin, Jasmine 2.5.2, and async tests. It appears to be the issue as reported here:

https://github.com/taras42/karma-jasmine-html-reporter/issues/5

I'm wondering if the 3.0 version of Jasmine and the plugin will resolve these issues, or if you think that this plugin needs further work to fully support async tests.

Thanks for any advice you can give me.

**Comment by konpikwastaken** [comment]
@andy-armstrong we had similar issues, but turns out the issues were with our tests - we had both 1) tests with side-effects (resetting page markup outside of fixtures setup, certain test order sequences would then fail) and 2) not all tests were using the async callback format jasmine expects (function callback from async test definition).

Have you checked & eliminated both of these possibilities as the cause within your code?

**Comment by dfederm** [comment]
@andy-armstrong Try what @konpikwastaken mentioned, and if you're still seeing issues try and share a minimal repro we can look at and debug.

**Comment by dfederm** [comment]
@konpikwastaken I was made an owner for the package and published v1.0.0. I'm abandoning this PR in favor of making my fork the primary repo now: https://github.com/dfederm/karma-jasmine-html-reporter

@andy-armstrong Feel free to continue the discussion over there if you're still seeing issues. Please provide a repro though if you do.

**Comment by andy-armstrong** [comment]
@dfederm Thanks for all your work to make this work. For some reason I didn't get notifications on your replies. We had put using the plugin on hold, but we've just upgraded to Jasmine 3 so we'll try again with your newly published version. Thanks again!

**Comment by srikanta2192** [comment]
I have commented all the test cases in my spec files, even then 'ng test' command results in 'appendChild of null' error.

**Pull request closed without merging**